src/main/java/org/seqdoop/hadoop_bam/FastaInputFormat.java

```diff
@@ -206,33 +226,29 @@ public FastaRecordReader(Configuration conf, FileSplit split) throws IOException
 		 */
 		private void positionAtFirstRecord(FSDataInputStream stream) throws IOException
 		{
-		    if (start > 0)
+			if (start > 0)
 			{
-			    stream.seek(start);
+				stream.seek(start);
 			}
 
-		    // we are now in a new chromosome/fragment, so read its name/index sequence
-		    // and reset position counter
-
-		    // index sequence
-		    LineReader reader = new LineReader(stream);
-		    int bytesRead = reader.readLine(buffer, (int)Math.min(MAX_LINE_LENGTH, end - start));
-
-		    current_split_indexseq = buffer.toString();
-		    // now get rid of '>' character
-		    current_split_indexseq = current_split_indexseq.substring(1,current_split_indexseq.length());
-		    
-		    // initialize position counter
-		    current_split_pos = 1;
-		    
-		    //System.err.println("read index sequence: "+current_split_indexseq);
-		    start = start + bytesRead;
-		    stream.seek(start);
-		    pos = start;
-		}
+			// we are now in a new chromosome/fragment, so read its name/index sequence
+			// and reset position counter
 
-		protected void setConf(Configuration conf)
-		{
+			// index sequence
+			LineReader reader = new LineReader(stream);
+			int bytesRead = reader.readLine(buffer, (int)Math.min(MAX_LINE_LENGTH, end - start));
+
+			current_split_indexseq = buffer.toString();
+			// now get rid of '>' character
+			current_split_indexseq = current_split_indexseq.substring(1,current_split_indexseq.length());
+
+			// initialize position counter
+			current_split_pos = 1;
+
+			logger.debug("read index sequence: {}");
+			start = start + bytesRead;
+			stream.seek(start);
+			pos = start;
```

Comment:
I disagree with implied premise that style fixes should fix a whole codebase or nothing at all (as opposed to, say, just the parts that are being touched in a given PR), but will take it out if you still feel that way, lmk.